schemaless/create_uuid_map.py

```diff
@@ -152,12 +157,12 @@ def values(self):
 class Node:
     def __init__(self,
                  record_id,
-                 date_opened=None,
+                 date=None,
                  parents=None,
                  children=None,
                  uuid=None):
         self.record_id = record_id
-        self.date_opened = date_opened
+        self.date = date
```

Comment:
I'm not sure what "date" is expected to be for a Node? "date_opened" feels clearer?

Reply:
different sources have different dates, eg TCO is date_issued. `date` here is just for choosing the most recent value for a given Node's parents/children